src/vm/eventtrace.cpp

```diff
@@ -6760,9 +6762,9 @@ VOID ETW::MethodLog::SendHelperEvent(ULONGLONG ullHelperStartAddress, ULONG ulHe
                                      ulHelperSize, 
                                      0, 
                                      methodFlags, 
-                                     NULL, 
+                                     W16_NULLSTRING,
```

Comment:
@sergig60, my concern is that there are likely to be many call sites that pass NULL for strings.  Your fix already ensures that if a NULL str is passed that it short circuits and never calls memcpy.  Thus, I don't think you need to pass an empty string.